upgrade/UPGRADE-v9.1.0.md

```diff
@@ -251,28 +251,28 @@ There you can find links to upgrade notes for other versions too.
             - `shopsys.router.localized.cache_dir`
 
 - added total count to frontend API connections ([#2141](https://github.com/shopsys/shopsys/pull/2141))
-    - see #project-base-diff to update your project
+    - see [project-base-diff](https://github.com/shopsys/project-base/commit/f250b822a01bd30e601b043a448f08c1698578d9) to update your project
 
```

Comment:
_random line_

I believe the change in `Shopsys\FrameworkBundle\Model\Feed\FeedExportFactory::create`
(lastseekid is now not typehinted) should be mentioned in upgrade notes as it's a BC break. What do you think?

introduced in #1922

Reply:
Nice catch! Added info to upgradenotes